nodejs/eks/examples/examples_test.go

```diff
@@ -131,13 +131,14 @@ func TestAccStorageClasses(t *testing.T) {
 	integration.ProgramTest(t, &test)
 }
 
-/*
 func TestAccCluster_withUpdate(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping test in short mode.")
+	}
 	test := getJSBaseOptions(t).
 		With(integration.ProgramTestOptions{
-			Dir:                  path.Join(getCwd(t), "./cluster"),
-			ExpectRefreshChanges: true,
```

Comment:
Why does this test no longer expect refresh changes?

Reply:
It still does, this was just removed in cleanup since we're already defining it in `getJSBaseOptions` down below

https://github.com/pulumi/pulumi-eks/blob/3535c657fecfdccc579faa4034fbf164815e66f2/nodejs/eks/examples/examples_test.go#L422-L429